app/models/dob.rb

```diff
@@ -0,0 +1,74 @@
+class Dob < Base
```

Comment:
Can we make this `DateOfBirthStep` for clarity?

Reply:
Yep